include/experimental/expected.hpp

```diff
@@ -1544,10 +1544,10 @@ class expected : public detail::expected_move_assign_base<T, E>,
   constexpr lvalue_reference_type operator*() & { return impl_base::val(); };
```

Comment:
Looks like this line should also remove the `;` too.

Reply:
done